Sources/AWSLambdaRuntimeCore/LambdaContext.swift

```diff
@@ -49,40 +50,118 @@ extension Lambda {
 extension Lambda {
     /// Lambda runtime context.
     /// The Lambda runtime generates and passes the `Context` to the Lambda handler as an argument.
-    public final class Context: CustomDebugStringConvertible {
+    public struct Context: BaggageContext.Context, CustomDebugStringConvertible {
+        private var storage: _Storage
+
+        final class _Storage {
+            var baggage: Baggage
+
+            let invokedFunctionARN: String
+            let deadline: DispatchWallTime
```

Comment:
Deadlines we'll eventually be able to handle in a distributed way -- so this will eventually move into the baggage.

See: Future: Deadlines and Cancelation patterns https://github.com/slashmo/gsoc-swift-baggage-context/issues/24